CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next version
 
 - Put your changes here...
```

Comment:
Delete this line. The one below removes the need for it.

Reply:
Done